17/alpine/hotspot/Dockerfile

```
@@ -1,6 +1,6 @@
-ARG JAVA_VERSION="17.0.8.1_1"
 ARG ALPINE_TAG=3.18.3
-FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-alpine AS jre-build
+ARG JAVA_VERSION=17.0.8.1_1
+FROM eclipse-temurin:"$JAVA_VERSION"-jdk-alpine AS jre-build
```

```suggestion
FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-alpine AS jre-build
```

To stay coherent with the Dockerfile already using an `ARG` (and it is a good practise)